framework/src/controller/schema/application_config_schema.js

```diff
@@ -148,6 +149,134 @@ module.exports = {
 					},
 					additionalProperties: false,
 				},
+				network: {
```

Comment:
maybe no need to nest the properties?

Reply:
Let's do it once we have chain also moved to the application to have a better understanding